File: js/sumPairs.js
exports.sumPairs = function(intArr, desiredNum) {
    let answer = []
    
    for (let i = 0; i < intArr.length; i++) {
        for (let j = i + 1; j < intArr.length; j++) {
            if (intArr[i] + intArr[j] == desiredNum) {
                answer.push([intArr[i], intArr[j]])
            }
        }
    }
    return !answer.length ? answer : 'unable to find pairs'
};

File: python/sum_pairs.py
def sum_pairs(int_arr, desired_num):
    answer = []

    for i in range(0, len(int_arr) - 1):
        for j in range((i + 1), len(int_arr)):
            if int_arr[i] +  int_arr[j] == desired_num:
                answer.append([int_arr[i], int_arr[j]])
    
    if len(answer):
        return answer
    else:
        return 'unable to find pairs'